pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -23,150 +23,165 @@ if [ "${LIMA_CIDATA_UDP_DNS_LOCAL_PORT}" -ne 0 ] || [ "${LIMA_CIDATA_TCP_DNS_LOC
-		pkgs="${pkgs} iptables"
-	fi
-	if [ "${LIMA_CIDATA_CONTAINERD_USER}" = 1 ]; then
-		if ! command -v newuidmap >/dev/null 2>&1; then
-			pkgs="${pkgs} shadow-utils"
+	elif command -v zypper >/dev/null 2>&1; then
+		pkgs=""
+		if [ "${LIMA_CIDATA_MOUNTTYPE}" = "reverse-sshfs" ]; then
+			if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ] && ! command -v sshfs >/dev/null 2>&1; then
+				pkgs="${pkgs} sshfs"
+			fi
+		fi
+		if [ "${INSTALL_IPTABLES}" = 1 ] && [ ! -e /usr/sbin/iptables ]; then
+			pkgs="${pkgs} iptables"
 		fi
-		if ! command -v mount.fuse3 >/dev/null 2>&1; then
+		if [ "${LIMA_CIDATA_CONTAINERD_USER}" = 1 ] && ! command -v mount.fuse3 >/dev/null 2>&1; then
 			pkgs="${pkgs} fuse3"
 		fi
-	fi
-	if [ -n "${pkgs}" ]; then
-		# shellcheck disable=SC2086
-		yum install ${yum_install_flags} ${pkgs}
-		yum-config-manager --disable epel >/dev/null 2>&1
-	fi
-elif command -v pacman >/dev/null 2>&1; then
-	pkgs=""
-	if [ "${LIMA_CIDATA_MOUNTTYPE}" = "reverse-sshfs" ]; then
-		if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ] && ! command -v sshfs >/dev/null 2>&1; then
-			pkgs="${pkgs} sshfs"
+		if [ -n "${pkgs}" ]; then
+			# shellcheck disable=SC2086
+			zypper --non-interactive install -y --no-recommends ${pkgs}
 		fi
-	fi
-	# other dependencies are preinstalled on Arch Linux
-	if [ -n "${pkgs}" ]; then
-		# shellcheck disable=SC2086
-		pacman -Sy --noconfirm ${pkgs}
-	fi
-elif command -v zypper >/dev/null 2>&1; then
-	pkgs=""
-	if [ "${LIMA_CIDATA_MOUNTTYPE}" = "reverse-sshfs" ]; then
-		if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ] && ! command -v sshfs >/dev/null 2>&1; then
-			pkgs="${pkgs} sshfs"
+	elif command -v apk >/dev/null 2>&1; then
+		pkgs=""
+		if [ "${LIMA_CIDATA_MOUNTTYPE}" = "reverse-sshfs" ]; then
+			if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ] && ! command -v sshfs >/dev/null 2>&1; then
+				pkgs="${pkgs} sshfs"
+			fi
 		fi
-	fi
-	if [ "${INSTALL_IPTABLES}" = 1 ] && [ ! -e /usr/sbin/iptables ]; then
-		pkgs="${pkgs} iptables"
-	fi
-	if [ "${LIMA_CIDATA_CONTAINERD_USER}" = 1 ] && ! command -v mount.fuse3 >/dev/null 2>&1; then
-		pkgs="${pkgs} fuse3"
-	fi
-	if [ -n "${pkgs}" ]; then
-		# shellcheck disable=SC2086
-		zypper --non-interactive install -y --no-recommends ${pkgs}
-	fi
-elif command -v apk >/dev/null 2>&1; then
-	pkgs=""
-	if [ "${LIMA_CIDATA_MOUNTTYPE}" = "reverse-sshfs" ]; then
-		if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ] && ! command -v sshfs >/dev/null 2>&1; then
-			pkgs="${pkgs} sshfs"
+		if [ "${INSTALL_IPTABLES}" = 1 ] && ! command -v iptables >/dev/null 2>&1; then
+			pkgs="${pkgs} iptables"
+		fi
+		if [ -n "${pkgs}" ]; then
+			apk update
+			# shellcheck disable=SC2086
+			apk add ${pkgs}
 		fi
 	fi
-	if [ "${INSTALL_IPTABLES}" = 1 ] && ! command -v iptables >/dev/null 2>&1; then
-		pkgs="${pkgs} iptables"
-	fi
-	if [ -n "${pkgs}" ]; then
-		apk update
-		# shellcheck disable=SC2086
-		apk add ${pkgs}
-	fi
+else
+	echo "LIMA_CIDATA_USE_DEFAULT_DEPENDENCY_RESOLUTION set, skipping regular dependency installation"
```

Comment:
Did you mean
```suggestion
	echo "LIMA_CIDATA_USE_DEFAULT_DEPENDENCY_RESOLUTION != 1, skipping regular dependency installation"
```

Reply:
Yeah, good catch. I think changing it to "skip" also makes more sense, like your other comment suggested.

The default value of "false" is easier to reason about, since most users won't interact with this option. Just pushed an update to address both comments, and also update the default example yaml